spmd/checkpoint.py

```diff
@@ -5,7 +5,6 @@
 import torch
 import torch.distributed.distributed_c10d as c10d
 from torch import distributed as dist
-from torch.distributed import ProcessGroup
```

Comment:
@kumpera did we remove some bindings from `torch.distributed.ProcessGroup`, and only leave them available in `c10d`?